pwa/src/templates/templateParts/formHeader/FormHeaderTemplate.module.css

```diff
@@ -4,14 +4,17 @@
   justify-content: space-between;
   gap: var(--gateway-ui-size-lg);
   margin-block-end: var(--gateway-ui-size-xl);
+  white-space: nowrap;
 }
 
 .container > * {
   flex: 1;
 }
 
 .title {
-  word-break: break-word;
+  font-size: var(--gateway-ui-font-size-2xl);
+  text-overflow: ellipsis;
+  overflow: hidden;
```

Comment:
It seems like these rules are not always applied (probably due to another rule from the denhaag package getting priority). 

I think we should update this selector from `.title` to `.container > .title`, this way it seems to always be used.